test/speed.c

```diff
@@ -6,6 +6,14 @@
 #include <stdint.h>
 #include <string.h>
 
+#if !defined(MPS2_AN386)
+#define SERIAL_MARKER() {\
+        hal_send_str("$");\
+    }
+#else
+#define SERIAL_MARKER()
+#endif
+
```

Comment:
maybe move this to the hal.h?

Reply:
good idea, originally thought that I should add a common header for tests (but that is ugly...)